### DISCUSSION TITLE
2024-05-28 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#320] [Meetings]
### Date: 2024-05-28, 3:00 – 4:00 pm EST
**Chair(s): Marjorie Rallins**

|**TIME** |**AGENDA TOPICS** |**LEAD(S)** |
| ----------- | -----| -----|
|3:00 - 3:05 | Welcome & opening remarks | April Lackey |
|3:05 - 3:10 | Update on user stories for Licensing Discussion|Tim Briscoe|
|3:10-3:30|UCUM 2.2 release process demo|Tim Briscoe|
|3:30-3:50|TBD: Annotations| Eza Hafeza, All |
|3:50 - 4:00 | TBD: New and Open Proposals (as time permits) | All
|4:00 | Adjourn | Eza Hafeza |

***OPEN ACTION ITEMS***
|**ITEM** |**DATE IDENTIFIED**|**TARGET COMPLETION**| **LEAD** |
| ----------- | -----| -----| -----|
| Continue asynchronous discussion on conformance / validation, [UCUM validation #250](https://github.com/orgs/ucum-org/discussions/250) | 2023-11-28 | Post first release | All |
| Collect use cases / user stories that may help inform license discussions | 2023-07-25 | 2023-11-14 | Tim Briscoe et. al. |
|Review the UCUM New Unit spreadsheet & update based on discussion at 4/23/2024 meeting|2024-04-23|2024-05-17|Eza Hafeza|
|Gather data on how frequent the use of core axes of system, method, analyte is in UCUM units|2024-03-26|2024-04-19| HDS Team|

***LINKS TO REFERENCE MATERIALS***
[UCUM Committee_Meeting Summary_2024-04-23_FINAL](https://docs.google.com/document/d/1joy9rDDo40_0IpD0VzP8qRWhi5dKRSf9/edit)

[UCUM New Unit Template_2024-01](https://docs.google.com/spreadsheets/d/1xBGaNLuMaJBFnFdi81ezDF5oHzN_Ozww/edit#gid=1763020506)

[All active GitHub issues/proposals](https://github.com/ucum-org/ucum/issues)

@ucum-org/regenstrief 
@ucum-org/advisors

**Comment by ucumworker** [comment]
-- Please reply above this line --

Hello Ucum-Org/ucum,

We received your email request and will get back to you as soon as we
can. Please know our response time does vary at different times of the
year due to our bi-annual release of LOINC and other events. 

Thank you,

Best Regards,
The Health Data Standards Team
Regenstrief Institute
